README.md

```diff
@@ -1,106 +1,31 @@
-
-### Code style
-
-We use [eslint](https://eslint.org/), [prettier](https://prettier.io/), typescript, and automatically set up git hooks to enforce
-these on commits. To avoid confusion, it is recommended to set this up your IDE.
-
-### Visual Studio Code
-
-Install the [eslint extension from the marketplace](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint). Configure your IDE to run `eslint --fix` on save (prettier will also reformat your code when doing this).
-
-### WebStorm and IntelliJ IDEA
-
-Configure your IDE to run `eslint --fix` on save (prettier will also reformat your code when doing this). It is also recommended to
-[set up Prettier as the default formatter](https://www.jetbrains.com/help/webstorm/prettier.html#ws_prettier_default_formatter).
-
-### CSS
-
-We use standard camelCase for classnames to enable linking of the stylesheets directly, improving simplicity in the process. Instead of using [BEM naming convention](http://getbem.com/naming/), we separate each Element and Modifier into separate names and use regular css (eks: `.accordion.open`) to access the combination.
-
-
-## Tests 🧨
-
-Run `yarn test` to run cypress tests in browser.
-
-Run `yarn coverage` to see pretty test coverage stats.
-
-Run 'yarn lint' to run lint checks
-
-## Building and deploying 🚚
-
-To create a distributable bundle, run `yarn build`. Environment variables set at build time will be baked into the bundle (e.g. `VITE_DEFAULT_LOCALE=en yarn build`). See the resulting `dist/index.html` for an example on how to load the build. At the moment you have to search for "/locales/${t.language}.json" and change it to "./locales/${t.language}.json".
-
-If the bundled files are to be served from a path other than the server root, you must pass the `--base=/path/to/folder/` argument to `yarn build`. The trailing slash is important.
-
-## Common problems in vs code
-1. Sometimes it's needed to restart eslint for it to work properly. E.g. When switching branches, eslint hangs sometimes. To fix this problem in vs code: Do the hot key for workbench.action.quickNaviagtePreviousInFilePicker and run command 'restart eslint server' or restart vs code.
-2. It's a common problem when writing reducers in rtk; invalid typescript-errors, prettier and lint-errors occur saying e.g. ',' is missing when in reality it's not. My suggestion is to restart vs code or ignore the error(if it's still possible to run the code). 
-
-
-## Run code with access-management backend
+## Step 2: Setup localtest to get test users
 
 To add test data to the app, do the following:
 
+- Clone following [repo](https://github.com/Altinn/app-localtest) and checkout branch feature/accessmanagement-subdomain(and for windows it's important to checkout commit 9ba9110e7934c4628076450d92bf829df8413619). Also remember to follow readme in that branch (Because localtest for podman doesn't work yet). You have completed this step when you can run localtest.
```

Comment:
```suggestion
- Clone following [repo](https://github.com/Altinn/app-localtest) and checkout branch feature/accessmanagement-subdomain(and for windows it's important to checkout commit 9ba9110e7934c4628076450d92bf829df8413619). Also remember to follow readme in that branch (Because localtest for podman doesn't work yet) and to follow the guide for running LocalTest locally. You have completed this step when you can run localtest using `dotnet run` from its src folder.
```